go/cmd/dolt/commands/filter-branch.go

```diff
@@ -118,6 +118,12 @@ func (cmd FilterBranchCmd) Exec(ctx context.Context, commandStr string, args []s
 		queryString = string(queryStringBytes)
 	}
 
+	if hasChanges, err := HasLocalChanges(ctx, dEnv); err != nil {
+		return HandleVErrAndExitCode(errhand.VerboseErrorFromError(err), usage)
+	} else if hasChanges {
+		verr := errhand.BuildDError("local changes detected, call dolt stash or commit changes").Build()
```

Comment:
(minor) suggesting "use" instead of "call", since this is always from a CLI context and "call" sounds SQLy to me. 
```suggestion
		verr := errhand.BuildDError("local changes detected, use dolt stash or commit changes before using filter-branch").Build()
```